### PR TITLE
feat: semantic-release

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ Data fetch:
   stage: "Data update"
   before_script:
     - yarn --frozen-lockfile
+    - yarn global add npx
```
Comment (lionelB) [inline .gitlab-ci.yml:29]: not sure you need to add it it seems it had been add with npm and co

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -47,12 +48,8 @@ Data fetch:
         # Commit files
         NOW=$(date +"%Y%m%d_%H%M")
         git add data
-        git commit -m "updates $NOW"
-        npm version patch
-        echo "Push to GIT"
-        git push origin HEAD:master
-        echo "Push to NPM"
-        npm publish --access=public
+        git commit -m "feat(data): $NOW update"
```
Comment (lionelB) [inline .gitlab-ci.yml:51]: does semantic release do the push to github too or you forgot to push ?

Comment (revolunet) [reply .gitlab-ci.yml:51]: en effet ca ne push pas !

**Comment by revolunet** [comment]
up :)

**Pull request merged**